AWS/aws-services.json

```diff
@@ -397,6 +397,10 @@
       {
         "resource": "Account",
         "resourceType": "AWS::Organizations::Account"
+      },
+      {
+        "resource": "ServiceControlPolicy",
+        "resourceType": "AWS::Organizations::ServiceControlPolicy"
```

Comment:
❓ The CloudFormation naming is like `AWS::Organizations::Policy` https://github.com/plerionhq/aws-org/blob/main/scp/AllAccountsBaselinePolicy.yml#L11 and then type `SERVICE_CONTROL_POLICY`. Do you think we should do something similar?

Reply:
i did think about that but i think users will find it `AWS::Organizations::ServiceControlPolicy` to be simpler than `AWS::Organizations::Policy`, which i think justifies the deviating from the CF naming convention. for instance if we go with `AWS::Organizations::Policy` then we immediately lose the fidelity of the asset type since as it stands we're only collecting SCPs, and we won't make that back up until (or _if_) we collect any of these: `TAG_POLICY | BACKUP_POLICY | AISERVICES_OPT_OUT_POLICY`.

Reply:
Our current use case is to find out who can access what based on their policies and actions defined. Currently, the way to determine which `resourceType` an `action` belongs to is from the resource definition that AWS has done in their doc. 

<img width="771" alt="Screenshot 2024-04-05 at 9 34 12 AM" src="https://github.com/plerionhq/plerion-asset-counter/assets/136443711/9705a699-1625-4ee5-96c0-060065ec6441">




We pull in a dataset that uses their doc as a source of truth. Having the names differ from the pattern will add a certain level of complexity but we can make an exception for certain resource types. We can go ahead with `AWS::Organizations::ServiceControlPolicy`